prowler/lib/persistence/__init__.py

```diff
@@ -0,0 +1 @@
+from .manager import *
```

Comment:
Is this a good practice? All the linters are flagging it and I think it is just to skip one part of the import path, right?